src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -463,6 +493,35 @@ private string ConfigureDatabase(InitializeViewRequestParams viewParams, Databas
                                 db110.DatabaseContainmentType = containmentTypeEnums[database.ContainmentType];
                             }
                         }
+                        if (prototype is DatabasePrototype130 db130)
+                        {
+                            if (databaseScopedConfigurationsCollection != null)
+                            {
+                                foreach (DatabaseScopedConfigurationsInfo dsc in database.DatabaseScopedConfigurations)
+                                {
+                                    foreach (DatabaseScopedConfiguration smoDscCollection in databaseScopedConfigurationsCollection)
+                                    {
+                                        if (smoDscCollection.Name == dsc.Name)
+                                        {
+                                            smoDscCollection.Value = dsc.ValueForPrimary == SR.prototype_db_prop_databasescopedconfig_value_fail_enabled
```

Comment:
Isn't this going to be a problem with the localized strings? Yet another reason to not use localized strings for the option values IMO